content/03-ventures/03-ideation-breakfast.md

```diff
@@ -0,0 +1,35 @@
+# Ideation Breakfast
+
+The purpose of the ideation breakfast is to discuss and decide on the execution of internal product ideas. Ideas can be submitted in the form of pitches to be evaluated at the ideation breakfast. After the ideation breakfast, the chosen product idea can be scheduled, planned and executed.
+
+## Formulated document
+
+Ideas are not presented in a spoken form but in a written pitch. The pitch needs to be written out in continuous text and should be self-explanatory without having further information. The structure can begin with a summary of the idea, followed by a description of the problem that led to the idea and then the solution. The deadline for submitting pitches is until the eve of each Ideation Breakfast. The deadline for submitting pitches is the evening before the ideation breakfast.
+
+[Example Pitch](https://www.notion.so/0727184bf5b840afa2fa4ea0c85ee864)
+
+## Read session
+
+Before we go into the discussion of ideas, we take the time to read all pitches from front to back. This helps to make sure, that everyone is on the same page, understands the product idea in detail and had time to think about it. Everyone can take notes and questions that can be addressed later.
+
+## Questions
+
+After all have read through the pitches, the author of the pitch has to answer open questions. The answers help to deepen the understanding of the product idea and diminish uncertainty.
+
+## Classify Impact / Effort
+
+In order to decide which product idea will make it into the next cycle, it is best to arrange the ideas according to impact and effort. The impact could include both economic potential and the impact of the problem.
+
+What the process looks like this:
+
+1. Draw an Impact / Effort Scale
```

Comment:
Regarding the whole impact/effort scale, it is more like a matrix or grid, so there are only four available options and no in-between:

- High impact / high effort
- Low impact / high effort
- High impact / low effort
- Low impact / low effort

See this example:
![impact-effort-matrix-1](https://user-images.githubusercontent.com/541628/53011950-ed48ca80-3441-11e9-97bd-73c2ffeeca1b.gif)

With this information in mind, we should re-think the seven steps you described below.

Reply:
That's true. I copied this out of Notion. Going to change some wording here.